Fabric/src/main/java/com/blamejared/crafttweaker/CraftTweakerFabric.java

```diff
@@ -38,6 +40,16 @@ public void onInitialize() {
         
         Services.PLATFORM.registerCustomTags();
         
+        CraftTweakerEvents.REGISTER_CUSTOM_BEP_EVENT.register((loader, regFun) -> {
```

Comment:
So on Fabric there's no `@EventListener` like on Forge?

Reply:
Correct, you also can't unregister events on fabric